package.json

```diff
@@ -23,6 +23,9 @@
     "request": "^2.58.0"
   },
   "devDependencies": {
+    "gulp": "^3.9.1",
+    "gulp-byo-jslint": "^1.1.1",
```

Comment:
I generally prefer to avoid submodules, because it adds an extra barrier for new contributors. I think the issues that caused you to create `gulp-byo-jslint` have been resolved in the original `gulp-jslint` project. Here's the output from a test project I just created.

![image](https://cloud.githubusercontent.com/assets/516559/18225259/8e3f7948-71a3-11e6-9879-ea7bb2696da3.png)

In fact, gulp-jslint even lets you bring your own version of jslint by passing a file path in `options.edition`, but again I'd prefer to just stick with the version that ships with the module. It also provides some stability by letting us pin to semver rather than cloning a Git repo that could have backwards-incompatible changes.